lambda-code/audit-logs-archiver/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM public.ecr.aws/lambda/nodejs:18 as builder
```

Comment:
It would take a bit more work, but if you use Alpine base images for this, it will cut down on a lot of the image vulnerabilities.

Here's an example of a Node.js Alpine based Docker image that works with Lambda:
https://github.com/cds-snc/scan-files/blob/main/module/s3-scan-object/Dockerfile

Reply:
I have been experimenting locally with an Alpine image based of NodeJS 18. It probably works once deployed in the actual AWS Lambda service but it does not in our Localstack implementation of it because it requires the AWS Lambda Runtime Interface to be supported for the use of https://github.com/aws/aws-lambda-nodejs-runtime-interface-client. 

It appears to be doable using this https://github.com/localstack/lambda-runtime-init but it complexify the setup. We could try to make it work in the future though.

Reply:
Will create a Github issue to remember it!